analysis/measures_definition.py

```diff
@@ -131,19 +144,29 @@
     measures.define_measure(
         name=f"has_appointments_{time_description}_weekly_age",
         numerator=has_appointment,
-        denominator=has_registration & (age > 18),
+        denominator=has_registration & (age >= 18),
         group_by={
             "age_greater_equal_65": age_greater_equal_65,
+            "age_65_69": age_65_69,
+            "age_70_64": age_70_64,
+            "age_75_79": age_75_79,
+            "age_80_84": age_80_84,
+            "age_greater_equal_85": age_greater_equal_85,
         },
         intervals=weeks(12).starting_on(start_date),
     )
 
     measures.define_measure(
         name=f"has_virtual_{time_description}_weekly_age",
         numerator=has_virtual_consultation,
-        denominator=has_appointment & has_registration & (age > 18),
+        denominator=has_appointment & has_registration & (age >= 18),
```

Comment:
yes, this looks like the change we discussed about, but I can see you didnt add the equal sign to all the denominators? was there a reason to only add it here?